project.mml

```diff
@@ -675,10 +683,16 @@ Layer:
           END AS oneway,
           CASE
             WHEN tags->'cycleway:left' IN ('track', 'opposite_track') THEN 'track'
+            WHEN tags->'sidewalk:left:bicycle' <> 'no' and tags->'sidewalk:left:segregated' = 'yes' then 'track'
```

Comment:
Minor (stylistic) comment but could we have an homogeneous SQL code convention here?

Almost all of CyclOSM SQL queries are written using `!=` operator (instead of `<>`) and uppercase keywords (`AND` / `THEN`). Thanks!